guacamole/src/main/webapp/app/navigation/directives/guacUserMenu.js

```diff
@@ -134,6 +134,48 @@ angular.module('navigation').directive('guacUserMenu', [function guacUserMenu()
                 return authenticationService.isAnonymous();
             };
 
+            /**
+             * Requests fullscreen for the Guacamole page.
```

Comment:
This function doesn't just request that the Guacamole page enter fullscreen mode, but toggles fullscreen.